src/library_vr.js

```diff
@@ -0,0 +1,227 @@
+var LibraryWebVR = {
+  $WebVR: {
+    DEVICE_TYPE_UNKNOWN: 0,
+    DEVICE_TYPE_HMD: 1,
+    DEVICE_TYPE_SENSOR: 2,
+
+    EYE_LEFT: 0,
+    EYE_RIGHT: 1,
+
+    initialized: false,
+    ready: false,
+    devices: [],
+    deviceHardwareIds: {},
+    nextHardwareDeviceId: 1,
+    getDevicesPromise: null,
+
+    selectedHMD: null,
+    selectedHMDId: 0,
+
+    init: function() {
+      if (WebVR.initialized)
+        return true;
+
+      WebVR.initialized = true;
+
+      if (!navigator.getVRDevices) {
+        WebVR.ready = true;
+        WebVR.devices = [];
+        return true;
```

Comment:
Is there any reason to `return true;` specifically in this function? I see nothing else is ever returned, and `emscripten_vr_init` is void anyways. I wonder if there should be a failure callback provided to C code if initialization is asynchronous?